modin/data_management/functions/default_methods/any_default.py

```diff
@@ -11,11 +11,22 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+# FIXME: This whole module is duplicating the logic of `default.py` and should be removed.
+
+"""Module houses default functions builder class."""
+
 from .default import DefaultMethod
 
 
 class ObjTypeDeterminer:
-    def __getattr__(self, key):
+    """
+    Class that routes work to the frame.
+
+    Provides an instance which forwards all of the `__getattribute__` calls
+    to an object under which `key` function is applied.
+    """
+
+    def __getattr__(self, key):  # noqa: D105
```

Comment:
`D105: missing docstring for magic method`, the logic of this method is trivial and already described in the class documentation

Reply:
I would rather we follow formal template, shouldn't be too hard.

Reply:
done